plugins/glustershd/init.go

```diff
@@ -35,6 +35,30 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Pattern:     "/volumes/{name}/heal/disable",
 			Version:     1,
 			HandlerFunc: glustershDisableHandler},
+		route.Route{
+			Name:        "GlustershInfo",
+			Method:      "GET",
+			Pattern:     "/volumes/{name}/{opts}/heal-info",
+			Version:     1,
```

Comment:
Please fill request and response struct fields for these rest routes wherever applicable. See [this issue](https://github.com/gluster/glusterd2/issues/584) for example and reference.